src/fidesops/api/v1/endpoints/health_endpoints.py

```diff
@@ -1,8 +1,7 @@
-from typing import Any, Dict
```

Comment:
🔍 good attention to detail!  looks like we're ignoring missing imports in mypy for some reason